### ISSUE TITLE
Can we save model based on val loss

### ISSUE THREAD

**Issue by Akshaysharma29** [flairNLP/flair#2268]
A clear and concise description of what you want to know.
Hi, team thanks for creating such a wonderful tool. I just want how to save the model based on Val loss.
Thanks

**Comment by alanakbik** [comment]
Hello @Akshaysharma29 as of Flair 0.9, this is now possible by setting the parameter `anneal_against_dev_loss` to `True` in the trainer: 

```python
import flair
from flair.datasets import UD_ENGLISH
from flair.embeddings import WordEmbeddings
from flair.models import SequenceTagger
from flair.trainers import ModelTrainer

flair.set_seed(123)

# 1. get the corpus
corpus = UD_ENGLISH().downsample(0.1)
print(corpus)

# 2. what label do we want to predict?
label_type = 'upos'

# 3. make the label dictionary from the corpus
label_dict = corpus.make_label_dictionary(label_type=label_type)
print(label_dict)

# 4. initialize embeddings
embeddings = WordEmbeddings('glove')

# 5. initialize sequence tagger
tagger = SequenceTagger(hidden_size=256,
                        embeddings=embeddings,
                        tag_dictionary=label_dict,
                        tag_type=label_type,
                        use_crf=True)

# 6. initialize trainer
trainer = ModelTrainer(tagger, corpus)

# 7. start training
trainer.train('resources/taggers/example-upos',
              learning_rate=0.1,
              mini_batch_size=32,
              max_epochs=25,
              patience=0,
              anneal_against_dev_loss=True,
              )
```
